src/components/common/cookies/cookies-controller.ts

```diff
@@ -27,22 +26,16 @@ export function cookiesPost(req: Request, res: Response): void {
     consentValue === "true" ? COOKIE_CONSENT.ACCEPT : COOKIE_CONSENT.REJECT
   );
 
-  createConsentCookie(res, consentCookieValue);
-
-  res.locals.backUrl = req.body.originalReferer;
-  res.locals.analyticsConsent = consentValue === "true";
-  res.locals.updated = true;
-  res.render("common/cookies/index.njk");
-}
-
-function createConsentCookie(
-  res: Response,
-  consentCookieValue: CookieConsentModel
-) {
   res.cookie(COOKIES_PREFERENCES_SET, consentCookieValue.value, {
     expires: consentCookieValue.expiry,
     secure: true,
-    httpOnly: true,
+    httpOnly: false,
     domain: res.locals.analyticsCookieDomain,
+    encode: String,
```

Comment:
Hi @andrew-moores - I've introduced an additional commit that extends the tests to assert that calls to `res.cookie` include the expected arguments. I initially had a bit of difficulty because the `expiry` property was changing between calls (by a second) and causing failures, but found I could assert for only the `httpOnly`, `secure` and `encode` properties by passing these as keys to an object passed to `sinon.match`